src/implementations/twig/components/file/__snapshots__/file.test.js.snap

```diff
@@ -79,13 +79,14 @@ exports[`File Thumbnail renders correctly 1`] = `
         (16.2 MB - PDF)
       </div>
       <a
-        aria-labelledby="#ecl-file-with-thumbnail-link #ecl-file-with-thumbnail-title #ecl-file-with-thumbnail-lang"
+        aria-labelledby="ecl-file-with-thumbnail-link ecl-file-with-thumbnail-title ecl-file-with-thumbnail-lang"
```

Comment:
the first id should be `ecl-file-with-thumbnail-link-label` instead of `ecl-file-with-thumbnail-link`